### PR TITLE
INTMDB:273: Fix replication_specs update error

### PR DESCRIPTION
**Pull request by abner-dou**
## Description

When update `replication_specs` property, the provider tried to send an empty id, so this fix gets the saved id and send to the API on change of this property

Also a minor changed to the `acceptance_test` naming (from `acceptances_test` -> `acceptance_test`)

Link to any related issue(s):[INTMDB-237](https://jira.mongodb.org/browse/INTMDB-273) https://github.com/mongodb/terraform-provider-mongodbatlas/issues/588

## Type of change:

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code

### CODE REVIEW

**Comment by themantissa** [comment]
@abner-dou  can you please add the issue to PR description?  You put the internal ticket but not the cited github issue so this PR won't show up there.

File: mongodbatlas/resource_mongodbatlas_cluster.go [hunk 1]
```diff
@@ -1179,10 +1180,8 @@ func expandReplicationSpecs(d *schema.ResourceData) ([]matlas.ReplicationSpec, e
 				original, _ := d.GetChange("replication_specs")
 				for _, s := range original.(*schema.Set).List() {
 					oldSpecs := s.(map[string]interface{})
-					if spec["zone_name"].(string) == cast.ToString(oldSpecs["zone_name"]) {
-						id = oldSpecs["id"].(string)
-						break
-					}
+					id = oldSpecs["id"].(string)
```
Comment (thetonymaster) [inline mongodbatlas/resource_mongodbatlas_cluster.go:1190]: if  `replication_specs` has only one element, shouldn't it have the max len set to `1` too?

**Review by thetonymaster — CHANGES_REQUESTED** [review]

File: mongodbatlas/resource_mongodbatlas_cluster.go (hunk 1 above)
Comment (abner-dou) [reply mongodbatlas/resource_mongodbatlas_cluster.go:1190]: this resource can have multiple replication specs per cluster per zone

**Comment by abner-dou** [comment]
Closed PR due is outdated

**Pull request closed without merging**